ocaml/testsuite/tests/typing-layouts-non-null-value/test_or_null.ml

```diff
@@ -115,37 +115,35 @@ Error: The layout of type string or_null is value, because
          of the definition of t at line 1, characters 0-40.
 |}]
 
-(* CR layouts v3.0: implement [non_null_immediate] *)
+(* CR layouts v3.0: implement [immediate_or_null] *)
 
 type t1 : non_null_value = string
 type t2 : non_null_value = int
-type t3 : non_null_immediate = int
-type t4 : value = int or_null
+type t3 : immediate = int
+type t4 : immediate_or_null = int or_null
```

Comment:
What's going on here? I understand we'll add this soon, but I'm surprised to see it in these tests.

Reply:
Right... but `immediate_or_null` is just an unknown identifier in the test. Or am I missing something?

Reply:
Yes, because I haven't added it yet. There's a `(* CR layouts v3.0: implement [immediate_or_null] *)` just above.

Reply:
Hmm... ok. I guess all of these tests are at the end of their respective blocks, and so the errors aren't masking others.